### ISSUE TITLE
Support mem_limit in docker-compose.yml

### ISSUE THREAD

**Issue by anthonyrisinger** [convox/rack#353]
```yaml
main:
  mem_limit: 1024m
  build: .
  ports:
    - 2000:2000
```

Something like that should be the same as a `convox deploy` followed by `convox scale main --memory=1024`

**Comment by bobzoller** [comment]
:+1:  my deploy process does exactly this, and it'd be nice to cut the rollout time in half during the (albeit few) deploys that include memory limit changes.  Also cpu?

**Comment by mwarkentin** [comment]
:+1: 

This is a valid `docker-compose.yml` value: https://docs.docker.com/compose/compose-file/#cpu-shares-cpu-quota-cpuset-domainname-hostname-ipc-mac-address-mem-limit-memswap-limit-privileged-read-only-restart-stdin-open-tty-user-working-dir

**Comment by ddollar** [comment]
Fixed in #1254
